### PR TITLE
fix: undefined in meeting url workflow variable

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Fixes issue that {MEETING_URL} workflow variable sometimes shows undefined in url. Like that: undefined/reschedule/...

### CODE REVIEW

**Review by joeauyeung — CHANGES_REQUESTED** [review]
I noticed that for workflows the event is typed as `BookingInfo`. Should we add the `bookerUrl` as a property so we ensure we're always handling org domains?

Found in `packages\features\ee\workflows\lib\reminders\smsReminderManager.ts` line 37

**Review by joeauyeung — DISMISSED** [review]
Feedback addressed. 

Just a heads up that we don't have to do `Omit<CalendarEvent, "bookerUrl">` if we're declaring `bookerUrl` in `ExtendedCalendarEvent`

File: packages/features/bookings/lib/handleSeats/types.d.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import type { Prisma } from "@prisma/client";
 import type z from "zod";
 
 import type { Workflow } from "@calcom/features/ee/workflows/lib/types";
-import type { AppsStatus } from "@calcom/types/Calendar";
+import type { AppsStatus, CalendarEvent } from "@calcom/types/Calendar";
```
Comment (CarinaWolli) [inline packages/features/bookings/lib/handleSeats/types.d.ts:5]: `CalendarEvent` wasn't imported, so `evt` was treated as `any` instead. Importing it now caused some other type errors that needed to be fixed

**Comment by CarinaWolli** [comment]
@joeauyeung Fixed the merge conflicts, can you re-review?

**Review by joeauyeung — APPROVED** [review]

**Pull request merged**